### ISSUE TITLE
Please add getLatLng method like google_maps_flutter

### ISSUE THREAD

**Issue by sboyd** [LuisThein/apple_maps_flutter#37]
Hello :)
First off thanks for using your time to build this package!! I really appreciate it.

**Is your feature request related to a problem? Please describe.**
I'm working on a project that has a requirement to allow users to draw an area on a map.
By following [this example](https://medium.com/@shadyboshra2012/flutter-free-hand-polygon-drawing-on-google-maps-796e8bd47e85) I was able to get it to work with the google maps package. What makes this possible is the getLatLng function of the controller. It takes screen coordinate and turns them into lat/long, but the apple_maps_flutter package does not.

**Describe the solution you'd like**
It'd be great if the apple_maps_flutter package had the same getLatLng function so I could allow users to draw on the map.

**Comment by bash-99** [comment]
> 

Agreed it will be nice if the package can provide access to getLatLng just like google maps package, I'm actually working on similar feature for ios device as explained by @sboyd  but i have no choice than to switch to google maps for now, hopefully this feature can be provided by apple_maps_flutter soon. Thanks

**Comment by bashcious** [comment]
> Hello :) First off thanks for using your time to build this package!! I really appreciate it.
> 
> **Is your feature request related to a problem? Please describe.** I'm working on a project that has a requirement to allow users to draw an area on a map. By following [this example](https://medium.com/@shadyboshra2012/flutter-free-hand-polygon-drawing-on-google-maps-796e8bd47e85) I was able to get it to work with the google maps package. What makes this possible is the getLatLng function of the controller. It takes screen coordinate and turns them into lat/long, but the apple_maps_flutter package does not.
> 
> **Describe the solution you'd like** It'd be great if the apple_maps_flutter package had the same getLatLng function so I could allow users to draw on the map.

Agreed it will be nice if the package can provide access to getLatLng just like google maps package, I'm actually working on similar feature for ios device as explained by @sboyd but i have no choice than to switch to google maps for now, hopefully this feature can be provided by apple_maps_flutter soon. Thanks
